includes/classes/Feature/Facets/Facets.php

```diff
@@ -470,7 +470,7 @@ public function build_query_url( $filters ) {
 		 */
 		$query_string = apply_filters( 'ep_facet_query_string', $query_string, $query_param );
 
-		$url        = $_SERVER['REQUEST_URI'];
+		$url        = isset( $_SERVER['REQUEST_URI'] ) ? sanitize_text_field( wp_unslash( $_SERVER['REQUEST_URI'] ) ) : '';
```

Comment:
I am not 100% sure about this change.

Reply:
@sjinks This change is pulled from upstream